.github/workflows/ci.yaml

```
@@ -234,8 +234,24 @@ jobs:
     name: Realm Server Tests
     runs-on: ubuntu-latest
     concurrency:
-      group: realm-server-test-${{ github.head_ref || github.run_id }}
+      group: realm-server-test-${{ matrix.testModule || github.head_ref || github.run_id }}
```

```suggestion
      group: realm-server-test-${{ matrix.testModule }}-${{ github.head_ref || github.run_id }}
```

I don’t think `||` makes sense here, each test file for a PR should be its own concurrency group… I think??

---

Oh I thought `||` in this context was a concat operator, no?

---

Ah, i found the documentation for these expressions--yes, i was using it incorrectly https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#format